### PR TITLE
NumeRe.NumeRe version 1.1.3.1

### PR DESCRIPTION
**Pull request by numeredev**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/23466)

### CODE REVIEW

File: manifests/n/NumeRe/NumeRe/1.1.3.1/NumeRe.NumeRe.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: NumeRe.NumeRe
+PackageVersion: 1.1.3.1
```
Comment (OfficialEsco) [inline manifests/n/NumeRe/NumeRe/1.1.3.1/NumeRe.NumeRe.installer.yaml:5]:
```suggestion
PackageVersion: 1.1.3.1-hf1
```

**Review by OfficialEsco — COMMENTED** [review]
The version number is different in this version, why is that?
(PackageVersion needs to match **A**dd or **R**emove **P**rograms)

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/23466#pullrequestreview-720468826

File: manifests/n/NumeRe/NumeRe/1.1.3.1/NumeRe.NumeRe.installer.yaml (hunk 1 above)
Comment (numeredev) [reply manifests/n/NumeRe/NumeRe/1.1.3.1/NumeRe.NumeRe.installer.yaml:5]: Hello @OfficialEsco!

Thanks for mentioning that. Will be fixed in installer in a few minutes. I'll commit an updated manifest with the correspondung SHA

**Review by OfficialEsco — APPROVED** [review]

**Review by quhxl — APPROVED** [review]

**Pull request merged**